s3_file_field/_multipart_minio.py

```diff
@@ -12,10 +12,22 @@ def __init__(self, storage: MinioStorage):
         # for pre-signing URLs when it exists
         self._signing_client: minio.Minio = getattr(storage, 'base_url_client', storage.client)
 
-    def _create_upload_id(self, object_key: str) -> str:
+    def _create_upload_id(
+        self,
+        object_key: str,
+        content_type: str = None,
+        content_disposition: str = None,
+    ) -> str:
+        metadata = {}
+        if content_type is not None:
+            metadata['Content-Type'] = content_type
+        if content_disposition is not None:
+            metadata['Content-Disposition'] = content_disposition
+        # Require content headers here
         return self._client._new_multipart_upload(
```

Comment:
This symbol `_new_multipart_upload` doesn't exist on the latest version of minio-py. It's been replaced with

```python
def _create_multipart_upload(self, bucket_name, object_name, headers)
```

Should we (or is it possible to) upgrade our dependency and use this newer method instead? @brianhelba

Reply:
@zachmullen No, we are pinned to minio-py v6 (the latest is v7), because v7 has a significantly different and incompatible API, which is not (yet) supported by django-minio-storage.